common-functions

```diff
@@ -116,23 +115,51 @@ service_backup() {
   trap 'rm -rf "$TMPDIR" > /dev/null' RETURN INT TERM EXIT
 
   (service_export "$SERVICE" > "${TMPDIR}/export")
-  docker run \
-      -e AWS_ACCESS_KEY_ID="$(cat "$AWS_ACCESS_KEY_ID_FILE")" \
-      -e AWS_SECRET_ACCESS_KEY="$(cat "$AWS_SECRET_ACCESS_KEY_FILE")" \
-      -e BUCKET_NAME="$BUCKET_NAME" \
-      -e BACKUP_NAME="${PLUGIN_COMMAND_PREFIX}-${SERVICE}" \
-      -v "${TMPDIR}:/backup" dokkupaas/s3backup:0.6.0
+
+  # Build parameter list for s3backup tool
+  BACKUP_PARAMETERS="-e AWS_ACCESS_KEY_ID=$(cat "$AWS_ACCESS_KEY_ID_FILE") \
+                     -e AWS_SECRET_ACCESS_KEY=$(cat "$AWS_SECRET_ACCESS_KEY_FILE") \
+                     -e BUCKET_NAME=$BUCKET_NAME \
+                     -e BACKUP_NAME=${PLUGIN_COMMAND_PREFIX}-${SERVICE} \
+                     -v ${TMPDIR}:/backup"
+
+  if [ -f "$BACKUP_CONFIG_ROOT/AWS_DEFAULT_REGION" ]; then
+    BACKUP_PARAMETERS="$BACKUP_PARAMETERS -e AWS_DEFAULT_REGION=$(cat "$BACKUP_CONFIG_ROOT/AWS_DEFAULT_REGION")"
+  fi
+
+  if [ -f "$BACKUP_CONFIG_ROOT/AWS_SIGNATURE_VERSION" ]; then
+    BACKUP_PARAMETERS="$BACKUP_PARAMETERS -e AWS_SIGNATURE_VERSION=$(cat "$BACKUP_CONFIG_ROOT/AWS_SIGNATURE_VERSION")"
+  fi
+
+  if [ -f "$BACKUP_CONFIG_ROOT/ENDPOINT_URL" ]; then
+    BACKUP_PARAMETERS="$BACKUP_PARAMETERS -e ENDPOINT_URL=$(cat "$BACKUP_CONFIG_ROOT/ENDPOINT_URL")"
+  fi
+
+  docker run "$BACKUP_PARAMETERS" dokkupaas/s3backup:0.6.0
```

Comment:
I'm not shure, but your lint tool wanted them. :)

see https://travis-ci.org/dokku/dokku-mariadb/jobs/264175402